content/tbb/tbb-31/contents.lr

```diff
@@ -3,7 +3,7 @@ _model: question
 title: Which platforms is Tor Browser available for?
 ---
 description:
-Tor Browser is currently available on [Windows](https://www.torproject.org/download/download-easy.html.en#windows), [Linux](https://www.torproject.org/download/download-easy.html.en#linux) and [macOS](https://www.torproject.org/download/download-easy.html.en#mac).
+Tor Browser is currently available on [Windows](https://www.torproject.org/download/#windows), [Linux](https://www.torproject.org/download/#linux) and [macOS](https://www.torproject.org/download/#mac).
```

Comment:
Maybe we can just add one link to https://www.torproject.org/download/ , because the anchors (#windows, #mac, etc) don't work.